src/test/article.spec.js

```diff
@@ -4,10 +4,12 @@ import sinon from 'sinon';
 import app from '../index';
 import BaseRepository from '../repository/base.repository';
 import { createUser, createArticle, generateArticle } from './utils/helpers';
+import { articleSample, articleSample2 } from './mockdata/articledata';
 import db from '../database/models';
 import helper from '../helpers/utils';
 
 const ARTICLES_API = '/api/v1/articles';
```

Comment:
There is no difference between `ARTICLE_API` and `articleEndpoint` can we use the first?

Reply:
Thank you for pointing that out 👍